Makefile

```diff
@@ -126,6 +126,7 @@ uninstall: manifests kustomize ## Uninstall CRDs from the K8s cluster specified
 	$(KUSTOMIZE) build config/crd | kubectl delete -f -
 
 deploy: manifests kustomize ## Deploy controller to the K8s cluster specified in ~/.kube/config.
+	kind load docker-image ${IMG} --name ${KIND_NAME}
```

Comment:
You should not add this on the deploy target.
make deploy is the usual command used to deploy an operator to a working cluster:

https://sdk.operatorframework.io/docs/building-operators/golang/tutorial/#2-run-as-a-deployment-inside-the-cluster

If you put this here, it assumes it's on a kind environment.

It's better if you add a new 'kind-load-operator-image' that does the kind load command and then you call this new target on the complete-kuttl-test

Reply:
Yes, you are right. I'll make the change. Thanks!